jdi-cucumber-test-generator-integration-tests/src/test/java/com/epam/test_generator/api/steps/given/Authentication.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.test_generator.api.steps.given;
+
+import com.epam.test_generator.api.steps.StepBackground;
+import com.epam.test_generator.controllers.user.request.LoginUserDTO;
+import cucumber.api.java.en.Given;
+
+public class Authentication extends StepBackground {
+
+    @Given("^I am Admin$")
+    public void logAdmin(){
+        LoginUserDTO loginUserDTO = new LoginUserDTO();
+        loginUserDTO.setEmail("admin@mail.com");
```

Comment:
Let's stop using magic strings. I suggest creating a final field to store "admin@mail.com" and reuse it,

Reply:
Constands created